### PR TITLE
APIGO-28709 - Cancel subscription with past due invoices + plan rebrand

### PR DESCRIPTION
**Pull request by cbordier**
Thank you for your contribution to the Amplify-Central repo.

## Describe the changes

Enter a brief description of your changes to communicate to the reviewers what you changed and why.

## Deploy preview link

Plan definition: http://pr-529.amplify-central.opendocs-builder.pcloud.axway.int/docs/manage_product_foundry/manage_product_plans/
Billing Integration: http://pr-529.amplify-central.opendocs-builder.pcloud.axway.int/docs/integrate_with_central/custom_billing/#invoice-past-due-flow

## Checklist for contributors

Before submitting this PR:

* Verify that all status checks have passed
* For more information on the Markdown linter rules, see [DavidAnson/markdownlint](https://github.com/DavidAnson/markdownlint)
* For first-time contributors, ensure that you have signed the [Axway CLA](https://cla.axway.com/)

### CODE REVIEW

**Comment by docsbuilder** [comment]
### <span aria-hidden="true">✅</span> Deploy preview is ready!

 |  Name | Link |
 |---------------------------------|------------------------|
 |<span aria-hidden="true">🔨</span> Latest commit | f7395b4 |
 |<span aria-hidden="true">😎</span> Deploy preview | http://PR-529.amplify-central.opendocs-builder.pcloud.axway.int |

**Comment by cbordier** [comment]
@lbadenhop, @georgiana-caras you can start review it.
This will need to be push before the release note since the release note will contain link to it.

File: content/en/docs/manage_product_foundry/manage_product_plans.md [hunk 1]
```diff
@@ -88,19 +90,41 @@ To configure a product plan from the create product wizard:
...
 
 To configure a product plan from the product detail:
 
 1. Navigate to the Plans section.
 2. Click **+ Add Plan** to open the plan creation wizard.
 3. Follow the plan wizard and enter the required information.
 
+### Billing information
+
+The billing information describe if the plan has financial aspect. It can be a Free plan or a Paid plan. The Free plan has no cost whereas the Paid plan has 2 kind of costs: a setup cost occurring at the creation of the subscription and a recurring cost happening at each billing cycle. Those field are optional: you can use one or the other or both. The Paid plan requires a currency so that the billing system will know how to bill the consumer.
```
Comment (robertengaxway) [inline content/en/docs/manage_product_foundry/manage_product_plans.md:103]: "The billing information describes if the plan has a financial aspect."
...
"Those fields are optional:"

File: content/en/docs/manage_product_foundry/manage_product_plans.md [hunk 2]
```diff
@@ -88,19 +90,41 @@ To configure a product plan from the create product wizard:
...
+
+You can also introduce if the subscription will be a never ended one or if the subscription will be automatically cancelled after a certain period of time (for instance a trial period before moving to a different plan). This is available for the Free and Paid plan.
+
+Lastly, for Paid plan, it is possible to setup an automatic subscription cancellation when the subscription has past-due invoices. Ie, if an invoice has not been paid in a timely manner, this will automatically cancel the subscription and terminate all associated access and credentials.
+
+To configure the plan billing, enter values for the following properties:
+
+* **Plan Type** - Free or Paid
+* **Currency** - only available for Paid plan - choose the currency that will be used to charge the subscription.
```
Comment (robertengaxway) [inline content/en/docs/manage_product_foundry/manage_product_plans.md:112]: Just a suggestion, but have a table with 3 columns, two are Free or Paid might make it easier to visualize

**Review by robertengaxway — APPROVED** [review]
Approved with a table and grammar suggestions.

**Review by georgiana-caras — APPROVED** [review]

**Review by lbadenhop — APPROVED** [review]

**Comment by cbordier** [comment]
All good to me.

**Pull request merged**